### DISCUSSION TITLE
mssql server

### DISCUSSION THREAD

**Discussion by euleabade** [jchristn/NetLedger#3] [General]
Would it be possible to use this application with SQL Server, if so what would be the level of security for data not being altered?

**Comment by jchristn** [comment]
Hi @euleabade thanks for your note.  Moving this to a discussion.

Yes, you can 100% use SQL Server as the backend for Netledger.  Netledger uses `WatsonORM` which has support for Sqlite, SQL Server, MySQL, and PostgreSQL.  A dependency update would likely be required, as I haven't updated this library in a few years (once I had built it to meet the needs I had for a particular project I had no need to udpate it).  

As far as security, the account you give to the library will need the ability to insert, select, delete, and update records.

Happy to help in any way I can.

Cheers, Joel
